catalogs/catalog-fileset/src/main/java/org/apache/gravitino/catalog/fileset/FilesetCatalogOperations.java

```diff
@@ -715,10 +701,6 @@ public String getFileLocation(NameIdentifier ident, String subPath, String locat
           case RENAME:
             // Fileset only mounts a single file, the storage location of the fileset cannot be
             // renamed; Otherwise the metadata in the Gravitino server may be inconsistent.
```

Comment:
the code comments also need to be removed?

BTW, this change breaks compatibility. We should also update the user documentation.

Reply:
done

Reply:
Not done. Code comments were not removed.